src/backend/server.js

```diff
@@ -76,54 +76,57 @@ async function issueDemoTokenHandler(req, res) {
         return;
     }
 
-    const value = web3.toWei(config.giveawayTokenAmounts, 'ether');
-    logger.info(`[blockchain] Issue demo token. Recipient: ${data.recipientAddress} value ${value} `);
-    try {
-        await tokenInstance.issue(data.recipientAddress, value, { from: config.account });
-        let issue = new Issue.model({
-            address: data.recipientAddress,
-            tokensAmount: config.giveawayTokenAmounts
+    let issue = new Issue.model({
+        address: data.recipientAddress,
+        tokensAmount: config.giveawayTokenAmounts
+    });
+    await issue.save().
```

Comment:
I think `await` is not necessary here. Please remove it or change `then().catch()` to `try/catch` block.